Resources/views/template/index.html

```diff
@@ -215,7 +215,16 @@
             var url = $(form).attr('action');
 
             //get form serialized data in order to remove matchedParams
-            var serializedData = $(form).serializeObject();
+//            var serializedData = $(form).serializeObject();
```

Comment:
if this line is not needed, delete it better!

Reply:
ok! I do it later!